packages/react/src/components/DateTimePicker/DateTimePickerV2.test.e2e.jsx

```diff
@@ -107,24 +136,25 @@ describe('DateTimePickerV2', () => {
 
     cy.findByText('2021-08-01 12:34 to 2021-08-06 10:49').should('be.visible').click();
 
-    cy.findByLabelText(i18n.startTimeLabel).type(
-      '{backspace}{backspace}{backspace}{backspace}{backspace}91:35'
-    );
+    // cy.get('#picker-test-start-time').clear();
+    cy.get('#picker-test-start-time').type('91:35');
+    cy.get('#picker-test-end-time').type('91:35');
     cy.findByText(i18n.applyBtnLabel).should('be.disabled');
 
-    cy.findByLabelText(i18n.startTimeLabel).type(
-      '{backspace}{backspace}{backspace}{backspace}{backspace}11:35'
+    // open start time time picker
+    cy.findByTestId('date-time-picker-time-btn-1').trigger('click');
+    // select hours
+    cy.findByTestId('date-time-picker-spinner-list-spinner-1').within(() =>
+      cy.findByText('01').click({ force: true })
+    );
+    cy.findByTestId('date-time-picker-time-btn-2').trigger('click');
+    cy.findByTestId('date-time-picker-spinner-list-spinner-1').within(() =>
+      cy.findByText('02').click({ force: true })
     );
     cy.findByText(i18n.applyBtnLabel).should('not.be.disabled');
 
-    cy.findByLabelText(i18n.endTimeLabel).type(
-      '{backspace}{backspace}{backspace}{backspace}{backspace}11:61'
-    );
+    cy.get('#picker-test-end-time').type('11:61 AM');
     cy.findByText(i18n.applyBtnLabel).should('be.disabled');
-
-    // set time to 11:00
-    cy.findByLabelText(i18n.endTimeLabel).type('{backspace}{backspace}00');
-    cy.findByText(i18n.applyBtnLabel).should('not.be.disabled');
```

Comment:
is there a reason we took this out?

Reply:
> is there a reason we took this out?

This is the behavior in <TimePickerSpinner> that if put 00 in the input, it is not a invalid value. Now we use <TimePickerDropDown> it would cause invalid date error.